File: QViz.NET/Periscope.QViz/JSON/UserResponse.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Response from User APIs
	/// </summary>
	public class UserResponse
	{
		/// <summary>
		/// Message returned from the User APIs
		/// </summary>
		public string message;
		
		/// <summary>
		/// User Object returned from the User APIs
		/// </summary>
		public User value;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public UserResponse()
		{
			this.message = "";
			this.value = new User();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/UserResponse.java
package com.periscope.qviz.json;

/**
 * JSON Model for Response from User APIs
 */
public class UserResponse {

	/**
	 * Message returned from the User APIs
	 */
	public String message;

	/**
	 * User Object returned from the User APIs
	 */
	public User value;

	/**
	 * Default Constructor
	 */
	public UserResponse() {
		this.message = "";
		this.value = new User();
	}

}